src/components/Home.js

```diff
@@ -39,60 +57,68 @@ export default function Home() {
-                  <li>
-                    <a href={'http://'} className="text is-2">
-                      Bitcoin
-                    </a>
-                  </li>
-                  <li>
-                    <a href={'http://'} className="text is-2 is-active">
-                      Start Learning
-                    </a>
-                  </li>
-                </ul>
-              </aside>
-            </div>
-          </div>
+
+      <section className={'section'}>
+        <div className={'container'}>
+          <h1 className={'title is-1'}>Education at core</h1>
+          <ul className={'columns mt3'}>
+            <li className={'column'}>
+              <figure className={'image is-128x128 is-centered'}>
+                <img src={codingImg} alt={'Coding'} />
+              </figure>
+              <h2 className={'is-2 has-text-centered mb10'}>Programmers</h2>
+              <p className={'text'}>
+                Build up your blockchain skills with interactive programming exercises and get certified automatically
+                at no fees.
+              </p>
+            </li>
+            <li className={'column'}>
+              <figure className={'image is-128x128 is-centered'}>
+                <img src={interactiveDiscussionImg} alt={'Discussion'} />
+              </figure>
+              <h2 className={'is-2 has-text-centered mb10'}>Universities</h2>
+              <p className={'text'}>
+                Courses are currated by blockchain scholars, built by and for Universities. Provides hands-on exercises
+                to your students.
+              </p>
+            </li>
+            <li className={'column'}>
+              <figure className={'image is-128x128 is-centered'}>
+                <img src={contentCreatorImg} alt={'Creator'} />
+              </figure>
+              <h2 className={'is-2 has-text-centered mb10'}>Companies</h2>
+              <p className={'text'}>
+                As a blockchain company, you can showcase your technology on an open source platform.
+                <br />
+                <Link to={'/contact'}> Contact us</Link> for featured tutorials.
+              </p>
+            </li>
+          </ul>
         </div>
       </section>
 
-      <div className={'container'}>
+      <div className={'container mt3'}>
         <section className="section has-background-color-darker">
           <div className="container">
             <div className="columns is-vcentered">
               <div className="column feature-box">
-                <h1 className={'title'}>Smart contracts</h1>
+                <h1 className={'title'}>Start Learning</h1>
                 <aside className="menu">
                   <ul className="menu-list">
                     <li>
-                      <a href={'http://'} className="text is-2">
-                        Functions
-                      </a>
+                      <Link to={'/Intro/Merkle_Tree'} className="text is-2">
```

Comment:
should be `<Link to={'/pages/Intro/Merkle_Tree'} className="text is-2">`

Reply:
fixed